packages/ui/molecules/Hero/Hero.twig

```diff
@@ -0,0 +1,143 @@
+{#
+/**
+ * @file
+ * Hero component.
+ *
+ * @param string $surtitle
+ * @param string $title
+ * @param string $subtitle
+ * @param array $ctas[]
+ * @param string $cta_url
+ * @param string $cta_label
+ *
+ * @param array                                     $attr
+ *   customize the root element attributes.
+ * @param array<Figure> $image
+ *   Image to display. See Figure.twig for the composition.
+ * @param array                                     $image_attr
+ *   customize the image element attributes.
+ * @param array                                     $content_attr
+ *   customize the content classes.
+ * @param 'left'|'center'|'right'                   $content_position
+ *   define the content position.
+ * @param array                                     $surtitle_tag
+ *   customize the tag of the surtitle element
+ * @param array                                     $surtitle_attr
+ *   customize the surtitle element attributes.
+ * @param array                                     $title_tag
+ *   customize the tag of the title element
+ * @param array                                     $title_attr
+ *   customize the title element attributes.
+ * @param array                                     $description_tag
+ *   customize the tag of the description element
+ * @param array                                     $description_attr
+ *   customize the description element attributes.
+ * @param array                                     $cta_attr
+ *   customize the cta element attributes.
+ */
+#}
+
+{% set attributes =
+  merge_html_attributes(attr ?? null, { class: ['hero', 'w-screen', 'h-screen', 'relative'] })
+%}
+
+{# Default images attributes #}
+{% set image_attributes =
+  merge_html_attributes(
+    image_attr ?? null,
+    {
+      class: ['hero__image', 'object-cover', 'w-full', 'h-full', 'absolute', 'z-under'],
+      src: src,
+      alt: alt|default(''),
+      width: width|default(false),
+      height: height|default(false)
+    }
+  )
+%}
+
+{# Default content attributes #}
+{% set content_position = content_position|default('center') %}
+{% set content_attributes =
+  merge_html_attributes(
+    content_attr ?? null,
+    { class: 'hero-content' },
```

Comment:
**nitpick:** The class should be `hero__content` 

The block here is called `hero`, everything else is an element in this block, the pattern for the class names should then be `hero__<element>`.

```suggestion
    { class: 'hero__content' },
```